build.sh

```diff
@@ -34,13 +51,23 @@ done
 
 SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-# Copy Thunder prject to Dockerfile context if project path is provided
+# Copy Thunder project to Dockerfile context if project path is provided
 if [[ "${PROJECT_PATH}" != "" ]]; then
-    cp --dereference --recursive "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+    if [[ "${OS_NAME}" == "osx" ]]; then
+        rm -rf "${SCRIPT_DIRECTORY}/www"
+        cp -R "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+    else
+        rm --recursive --force  "${SCRIPT_DIRECTORY}/www"
+        cp --dereference --recursive "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+    fi
```

Comment:
While I am always in favor of using long-form arguments, I would vote for just using the short-hand syntax which works in all cases and we would get of all the if/else code complexities.